### ISSUE TITLE
Admin Toolbar has double text

### ISSUE THREAD

**Issue by SConstantinou** [ironmansoftware/universal-dashboard#1469]
#### Describe the Issue
Admin Toolbar shows the Text 2 times

#### To Reproduce
check attached screenshot

#### Expected behavior
Text should be shown only once

#### Screenshots
![Annotation 2020-01-22 172207](https://user-images.githubusercontent.com/35262217/72906979-c96dc300-3d3b-11ea-9507-c6cba9dfacba.png)

#### Version Information
- **Operating System**: Windows Server 2019 Core
- **PowerShell Version**: Windows Powershell 5.1
- **Universal Dashboard Version**: 2.8.2
- **UD Hosting Method**: IIS

**Comment by adamdriscoll** [comment]
Are you just using the default theme?

**Comment by SConstantinou** [comment]
Yeah just the default theme

Thank you
Stephanos Constantinou
www.sconstantinou.com

**Comment by e1ze** [comment]
i am having the same problem also using the default theme (customized)

**Comment by adamdriscoll** [comment]
Working on a fix for this. Thanks for the heads up. Trying to get a quick turn around on 2.8.3.

**Comment by adamdriscoll** [comment]
Can either of you provide some more info? I'm trying to reproduce and haven't be able to yet. From the screenshot, it looks like it's using a fixed nav bar so I thought it was that but that doesn't seem to do it for me. 

```
        $NavBar = New-UDSideNav -Content {
            New-UDSideNavItem -Text "Hello" -Url ""
            New-UDSideNavItem -Text "Hello2" -Url ""
        } -Fixed

        $Dashboard = New-UDDashboard -Title "Test" -Content {
            New-UDElement -Tag 'div' -Endpoint {
                New-UDElement -Tag 'div' -Id 'dynamic' -Endpoint {

                }
            }
        } -Navigation $NavBar
```

**Comment by SConstantinou** [comment]
Please check below path of the code

```
$Logo = New-UDImage -Path "$PSScriptRoot\Logo.png" -Height 50 -Width 153
$Footer = New-UDFooter -Copyright 'Copyright © 2019 xxxxxxxxxx Ltd'
$InitialEndPoints = New-UDEndpointInitialization -Module @("$PSScriptRoot\Dash.psm1") -Variable @('Credentials')

$Authentication = New-UDAuthenticationMethod -AppId 'xxxxxxxx-xxxxx-xxxxx-xxxxxxxx-xxxxxxx' -Instance 'https://login.microsoftonline.com' -Domain 'xxxxxxxxxxx.onmicrosoft.com' -TenantId 'xxxxxxxx-xxxxxx-xxxx-xxxx-xxxxxxxxxxxxxxxxx'

$AuthorizationPolicy = New-UDAuthorizationPolicy -Name AccessPolicy -Endpoint {
    param($User)

    $User.HasClaim('groups','xxxxxx-xxxxx-xxxxxxxx-xxxxxx-xxxxxxxxxxxx')
}

$LoginPage = New-UDLoginPage -AuthenticationMethod $Authentication -AuthorizationPolicy $AuthorizationPolicy -Logo $Logo

$NavigationBar = New-UDSideNav -Content {
    New-UDSideNavItem -Text "Home" -PageName "Home" -Icon home
    New-UDSideNavItem -Text "Cloud" -Children {
        New-UDSideNavItem -Text "O365 Licenses" -PageName "O365 Licenses" -Icon id_badge
        New-UDSideNavItem -Text "Exchange Online" -PageName "Exchange Online" -Icon exchange
    } -Icon cloud
    New-UDSideNavItem -Text "On Premise" -Children {
        New-UDSideNavItem -Text "Active Directory" -PageName "Active Directory" -Icon dice_d20
        New-UDSideNavItem -Text "DNS" -PageName "DNS" -Icon globe
        New-UDSideNavItem -Text "DHCP" -PageName "DHCP" -Icon sitemap
        New-UDSideNavItem -Text "Server Status" -PageName "Server Status" -Icon server
        New-UDSideNavItem -Text "Computer Status" -PageName "Computer Status" -Icon laptop
    } -Icon building
} -Fixed

$Pages = @($HomePage,$ADStatsPage,$DNSPage,$DHCPPage,$ServerStatusPage,$AllServersStatusPage,$LicensesPage,$ExchangeOnlinePage,$ComputerStatusPage,$AllComputersStatusPage)
$Dashboard = New-UDDashboard -Title IRTDashboard -Pages $Pages -EndpointInitialization $InitialEndPoints -Navigation $NavigationBar -NavBarLogo $Logo -Footer $Footer -LoginPage $LoginPage
Start-UDDashboard -Wait -Dashboard $Dashboard -Certificate $Certificate -Port 80 -HttpsPort 443 -AdminMode
```

**Comment by adamdriscoll** [comment]
Resolved in tonight's build.

**Comment by SConstantinou** [comment]
I still have the issue

**Comment by SConstantinou** [comment]
Resolved with 2.9.0
